.github/workflows/developer.yml

```diff
@@ -92,7 +92,7 @@ jobs:
         mkdir build
         doxygen --version
         cd build
-        cmake .. -DENABLE_DOCS=On -DJasper_ROOT=~/Jasper -DCMAKE_PREFIX_PATH="~/bacio;~/w3emc" -DCMAKE_Fortran_FLAGS="-g -fprofile-abs-path -fprofile-arcs -ftest-coverage -O0 -Wall -fsanitize=address" -DCMAKE_C_FLAGS="-g -fprofile-abs-path -fprofile-arcs -ftest-coverage -O0 -Wall -fsanitize=address" -DFTP_TEST_FILES=ON -DCMAKE_BUILD_TYPE=Debug 
+        cmake .. -DENABLE_DOCS=On -DJasper_ROOT=~/Jasper -DCMAKE_PREFIX_PATH="~/bacio;~/w3emc" -DCMAKE_Fortran_FLAGS="-g -fprofile-abs-path -fprofile-arcs -ftest-coverage -O0 -Wall -fno-omit-frame-pointer -fsanitize=address" -DCMAKE_C_FLAGS="-g -fprofile-abs-path -fprofile-arcs -ftest-coverage -O0 -Wall -fno-omit-frame-pointer -fsanitize=address" -DFTP_TEST_FILES=ON -DCMAKE_BUILD_TYPE=Debug 
```

Comment:
Is this related to #380 or #329?

Reply:
The -fno-omit-frame-pointer causes more info to be printed when the address sanitizer finds a problem.

We should be using it in every developer.yml file, so if you are having some santizer failure, and you see it is not there, go ahead and add it.

It causes the error message to contain line numbers for the stack trace. Very helpful!